### ISSUE TITLE
StemSnapH / StemSnap V in static .otf

### ISSUE THREAD

**Issue by luke-snider** [googlefonts/fontmake#946]
When building static .otf files from .ufo sources I experienced the following:

If I only set a single value for StemSnapH / StemSnapV these somehow get lost in the .otf.
When setting 2 values in StemSnapH / StemSnapV these end up in the .otf.
Is this on purpose?

Another thing I encountered was that if these values are in the wrong order (not increasing, but decreasing), they also get lost. 
Of course a decreasing order is not according to the PS specification, but maybe it would make sense to let fontmake set the order to increasing?

**Comment by anthrotype** [comment]
Hi, I'm not very familiar with PS hinting. I think ufo2ft is responsible for setting those here:

https://github.com/googlefonts/ufo2ft/blob/6d618f6ee97108228aa8af9f43a184285656ac55/Lib/ufo2ft/outlineCompiler.py#L1366-L1371

There's a comment saying "only write the stems if both (stemSnapH and stemSnapV) are defined" which seems to hint at what you found, so it appears to be deliberate.

Does this answer you questions?

> Another thing I encountered was that if these values are in the wrong order (not increasing, but decreasing), they also get lost.

I don't know where that occurrs, maybe in fontTools.cffLib.

**Comment by luke-snider** [comment]
Thank you!
But what happend to me is that:
1 Value for stemSnapH and 1 Value for stemSnapV
is set - so both are defined.

**Comment by anthrotype** [comment]
yeah, that's weird. Could you upload a minimal UFO source that reproduce the issue?
